### PR TITLE
WIP : Use of ORTB library for request generation.

### PR DESCRIPTION
**Pull request by pm-kapil-tuptewar**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/pubmaticBidAdapter.js [hunk 1]
```diff
@@ -1096,324 +597,85 @@ export const spec = {
...
-    if (window.location.href.indexOf('pubmaticTest=true') !== -1) {
-      payload.test = 1;
-    }
-
-    // adding schain object
-    if (validBidRequests[0].schain) {
-      deepSetValue(payload, 'source.ext.schain', validBidRequests[0].schain);
-    }
-
-    // Attaching GDPR Consent Params
```
Comment (jlquaccia) [inline modules/pubmaticBidAdapter.js:1269]: In regard to the buildRequests function when using the ortbConverter library, i'm assuming that the ortbConverter lib handles properly setting kv's for privacy related params like gdpr, ccpa, gpp, coppa, etc.?  i'm guessing that this is all just abstracted away now?

**Comment by jlquaccia** [comment]
hey @kapil-tuptewar, took some time to go through this PR.  overall, i think it looks pretty good!  just left one question up above.

File: modules/pubmaticBidAdapter.js (hunk 1 above)
Comment (pm-kapil-tuptewar) [reply modules/pubmaticBidAdapter.js:1269]: Yes thats correct @jlquaccia. This library is taking care of all these stuff.

Comment (jlquaccia) [reply modules/pubmaticBidAdapter.js:1269]: @kapil-tuptewar nice!  ok, that's what my assumption was, just wanted to confirm.  cool, all LGTM then.

**Review by jlquaccia — APPROVED** [review]

**Pull request closed without merging**